Formula/qwerty-fr.rb

```diff
@@ -0,0 +1,40 @@
+class QwertyFr < Formula
+  desc "Qwerty keyboard layout with French accents"
+  homepage "https://qwerty-fr.org"
+  url "https://github.com/qwerty-fr/qwerty-fr/releases/download/v0.7/qwerty-fr_0.7_mac.zip"
+  sha256 "34bc8c8f764798ed7e48a3f2404319e09a6d8bfcbef350b25df32387a3199bfa"
+  license "MIT"
+
+  def install
+    # Homebrew automatically extracts the folder inside the archive to the cur directory (not configurable)
+    # so we need to rebundle it
+    mkdir "qwerty-fr.bundle"
+    mv "Contents", "qwerty-fr.bundle/"
+    prefix.install "qwerty-fr.bundle"
+  end
+
+  def caveats
```

Comment:
This needs to be in the upstream documentation since it doesn't seem homebrew specific.

Reply:
The linking requirement is homebrew-specific.

If I remove the restart note & the layout selection help, how will users learn of that if they install the package straight from `brew`?

I'm worried my repo may receive a lot invalid issues from people who didn't find this piece of information easily enough.